producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
```

Comment:
I think we should continue using markdown for documentation, we shouldn't have multiple formats in the same project.

Reply:
I don't see a guideline on using MD vs adoc. I picked the adoc because it is more flexible and has a lot more benefits. For all new products in drivers we were using adoc, we were even planning to migrate java driver documentation to adoc. The doc team also prefers this format.